### ISSUE TITLE
Still maintained? 

### ISSUE THREAD

**Issue by quentin23soleil** [adrielcafe/voyager#493]
It seems issues are growing and we had no new release since June 5th, which makes me believe this library is not maintained anymore. If so, it might be good to disclose it on the readme and/or archive it.

**Comment by YoussefHachicha** [comment]
yap this is concerning

**Comment by nvkleban** [comment]
Agree, at least some level communication from maintainers on what are the plans, is it still maintained or not, would be very helpful.
I've asked in June in the [discussion](https://github.com/adrielcafe/voyager/discussions/449) if any help needed with any specific part, still no answer.

**Comment by akardas16** [comment]
Even the basics are broken. There is still no one who cares about this library

**Comment by acmpo6ou** [comment]
I agree, I wanted to contribute to the docs, I wanted to improve the grammar. And, I saw there was an open discussion about it already, but there was no activity. It does seem like the devs are very busy with personal life stuff or something.

**Comment by Deorigami** [comment]
Time to move on guys, Use Official navigation from Jetbrains

**Comment by Deorigami** [comment]
me myself planning on migration

**Comment by quentin23soleil** [comment]
> the devs are very busy with personal life stuff or something

and that'd be totally fine and understandable obviously. And they've helped the community so greatly I'm really grateful, I didn't meant this issue to sound anything near unpleasant.

**Comment by Syer10** [comment]
New beta is out 😄  https://github.com/adrielcafe/voyager/releases/tag/1.1.0-beta03

**Comment by nvkleban** [comment]
I received a response from the maintainer, and I hope he don’t mind me sharing it here.

>Me: Hi! My name is Nick, I’m Android Developer and active user of the Voyager. I really appreciate you work on this library, but the question that I and other users have - is Voyager still supported, is there any plans on it?
> It seems the original maintainer is no longer interested in this project, and you probably don’t have much spare time too.

> DevSrSouza : Yeh, that is the case most of the time I don't have spare time to it, honestly my mind have not being very good this last months, but I still look for Voyager when the time comes. So it is still supported. But because I'm currently the only maintainer, I kinda shift what I would be focusing, so, in practice, what I have being focusing is the Core of the library, make sure is Extensible and work as expected. This means that, extensions that we have like TabNavigator, BottomSheetNavigator, etc, I don't have enough time to focus, so we let the community build their on implementations.
> At the new docs, we move it to github, we appreciate every type of contribution, even docs. We have a page focus on community projects that is extending voyager and a FAQ that contains links to some snippets to try out, for example, Swipe Back on iOS. But this APIs, if I would put into the library, it come with a cost of maintaining and implementing very well, and honestly, because I don't have much spare time currently, I can't put this so asked APIs inside the library. But I guarantee to have enough API in Voyager Core Navigator to you be able to extend and implement it easily.

**Comment by acmpo6ou** [comment]
@nvkleban It's pretty sad there is only one guy working on this, I wish more people could help out maintain this. Otherwise the library is gonna be stagnating.

**Comment by hristogochev** [comment]
Hello!

I love Voyager and I just published a [fork](https://github.com/hristogochev/vortex) with rewritten internals that fixes a lot and adds some things. 
You may find it useful if you don't use ScreenRegistry, Hilt, LiveData or RxJava.

There are some API changes but it has mostly the same API as Voyager.
I also renewed the [documentation](https://hristogochev.github.io/vortex/) so you can hopefully more easily adapt to the changes!

I plan to maintain it in the long term and increasingly add new features.

Long live Voyager! 🎉

**Comment by kevinvanmierlo** [comment]
@hristogochev I don't want to give up on Voyager just yet. But you do have some nice changes compared to Voyager. I was struggling with disposing and remembering tabbar screens when navigating away in the navigator above the tabbar. I think by attaching the navigator to a screen that it will probably just work. The transitions also look nice. And the bottom sheet was not always working correctly, by simplifying it, that also seems to be solved. However, I don't like that you forced the default navigation saver. It's fine that it's the default, but I prefer passing objects to the next screen without making them serializable or something. I created a custom saver for Voyager which just saves the list to a Map and it will just revert back to the initial screen when destroyed. Also is there a way to talk in that repo? Issues are of course turned off because it's a fork. I hope some of these changes just end up in Voyager, but in the meantime. 

I'm not sure what the maintainers think about simplifying the code like you did. But that seems to be the right direction for me to solve a lot of hard to solve issues. Also make it easier to maintain and just let the community create it's own extensions around it. Also I'd like the library to have more "open" stuff. Some stuff like the TabNavigator I had to remake everything because it was not working for me. If it was open, I could've solved it more easily.

**Comment by hristogochev** [comment]
> @hristogochev I don't want to give up on Voyager just yet. But you do have some nice changes compared to Voyager. I was struggling with disposing and remembering tabbar screens when navigating away in the navigator above the tabbar. I think by attaching the navigator to a screen that it will probably just work. The transitions also look nice. And the bottom sheet was not always working correctly, by simplifying it, that also seems to be solved. However, I don't like that you forced the default navigation saver. It's fine that it's the default, but I prefer passing objects to the next screen without making them serializable or something. I created a custom saver for Voyager which just saves the list to a Map and it will just revert back to the initial screen when destroyed. Also is there a way to talk in that repo? Issues are of course turned off because it's a fork. I hope some of these changes just end up in Voyager, but in the meantime.
> 
> I'm not sure what the maintainers think about simplifying the code like you did. But that seems to be the right direction for me to solve a lot of hard to solve issues. Also make it easier to maintain and just let the community create it's own extensions around it. Also I'd like the library to have more "open" stuff. Some stuff like the TabNavigator I had to remake everything because it was not working for me. If it was open, I could've solved it more easily.

My apologies, I didn't notice that issues are not enabled! I've just enabled them and you can report any issues and suggest things there.

**Comment by nvkleban** [comment]
@hristogochev our solution is probably would be just to migrate to Android Jetpack Navigation, it had some improvements during the past year and finally has some type safety. I actually considered migrating to your fork, but unfortunately we using hilt and screen registry quite extensively.

**Comment by hafiz013** [comment]
@hristogochev  don't give up or please open to public to enhanced this library. so far this library can't support 2.1.0 kotlin that effected ios which i think most most most important. Any idea how to open this project to public to maintain it? thanks hopefully can get patch for 2.1.0 for ios thanks

**Comment by hristogochev** [comment]
> [@hristogochev](https://github.com/hristogochev) don't give up or please open to public to enhanced this library. so far this library can't support 2.1.0 kotlin that effected ios which i think most most most important. Any idea how to open this project to public to maintain it? thanks hopefully can get patch for 2.1.0 for ios thanks

The fork is open to public, you can make issues and pull requests. It's also compatible with Kotlin 2.1.0. I do not plan to make pull requests to Voyager since the internals of the library are essentially rewritten.

**Comment by hafiz013** [comment]
@hristogochev you can try the beta 3 still not working on org.jetbrains.kotlin.multiplatform: 2.1.0 or 2.1.10 you can check here : 
https://github.com/adrielcafe/voyager/issues/522
